src/util/features/transparentVideo/createTransparentVideoTag.js

```diff
@@ -0,0 +1,26 @@
+/**
+ *
+ * @description Creates a hidden HTMLVideoElement with the spcified videoOptions
+ * @param {{autoplay, playsinline, loop, muted, poster, src }} videoOptions
+ * @return {HTMLVideoElement}
+ */
+function createTransparentVideoTag(videoOptions) {
+  let { autoplay, playsinline, loop, muted, poster, src } = videoOptions;
+
+  let el = document.createElement('video');
+  el.style.visibility = 'hidden';
```

Comment:
seeThru will act on this video element, it should not be visible (in fact seeThru marks it as invisible when it starts working).
I wanted to prevent a possible UI jitter, though very unlikely to happen.